src/main/java/com/google/edith/servlets/UserInsights.java

```diff
@@ -21,6 +21,7 @@
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Comparator;
+import java.util.Collections;
```

Comment:
is collections used? otherwise can delete

Reply:
Done.